googletest/cmake/internal_utils.cmake

```diff
@@ -94,6 +94,7 @@ macro(config_compiler_and_linker)
     set(cxx_no_exception_flags "-EHs-c- -D_HAS_EXCEPTIONS=0")
     set(cxx_no_rtti_flags "-GR-")
   elseif (CMAKE_COMPILER_IS_GNUCXX)
+    message("in here")
```

Comment:
I think it may be better to say message ("CMAKE COMPILER IS GNUCXX")

Reply:
Oh, sorry, I wasn't planning to land any of the message()s, I was just trying to figure out why I had to manually pass CMAKE_COMPILER_IS_GNUCXX since it seemed like the sort of thing that should be automatic. I'll revert them out.